oom_kill/README.md

```diff
@@ -39,6 +39,29 @@ In addition, Windows, Container-Optimized OS, and CentOS/RHEL versions earlier t
 
 3. [Restart the Agent][3].
 
+### Configuration with Docker
+
+In addition to mounting `system-probe.yaml` and `oom_kill.d/conf.yaml` as described above, do the following configuration:
+
+1. Mount the following volumes to the Agent container:
+
+    ```
+    -v /sys/kernel/debug:/sys/kernel/debug 
+    -v /lib/modules:/lib/modules 
+    -v /usr/src:/usr/src
+    ```
+
+2. Add the following permission to enable BPF operations:
+
+    ```
+    --privilege
+    ```
+    
+    In kernel version 5.8 and greater, `CAP_BPF` works with the `--cap-add` feature.
```

Comment:
do you want to say that from the linux kernel `5.8`, the user can use `--cap-add CAP_BPF` instead of `--privilege` ? 

if we maybe the sentences can be more explicit
```suggestion
    From kernel version 5.8, the `--privilege` parameter can be replaces by `--cap-add CAP_BPF`. ```

Reply:
Thanks, yes makes sense. 
I commit it but removed "```" in the end as it looks unintended.

Reply:
@clamoriniere can you review?